tests/foreman/virtwho/cli/test_nutanix.py

```diff
@@ -181,7 +181,7 @@ def test_positive_prism_central_deploy_configure_by_id_script(
     ):
         """Verify "hammer virt-who-config deploy" on nutanix prism central mode
 
-        :id: 224ad753-6186-4b09-a72c-458839a8e412
+        :id: e85d7c19-ecf7-4ddd-a66b-a429e18fdc23
```

Comment:
@yanpliu Could you revert this change for the test case ids? it seems irrelevant.

Reply:
@Gauravtalreja1 Have already done.